packages/server/src/auth/auth.service.ts

```diff
@@ -0,0 +1,82 @@
+import { Injectable } from '@nestjs/common';
+import * as bcrypt from 'bcrypt';
```

Comment:
Use argon2 instead of bcrypt

Reply:
what's the motivation for using argon2 instead of bcrypt?